misc/cf-support

```diff
@@ -182,6 +182,41 @@ if [ -f $WORKDIR/share/cf-support-nova-hub.sh ]; then
   source $WORKDIR/share/cf-support-nova-hub.sh
 fi
 
+echo "Analyzing CFEngine core dumps"
```

Comment:
This should also handle systems that have `coredumpctl` available.

Reply:
ack